cluster-agent/values.yaml

```diff
@@ -3,14 +3,22 @@ deployment:
   image:
     repository: docker.io/appdynamics/cluster-agent
     tag: latest
-    pullPolicy: IfNotPresent
+    pullPolicy: Always
```

Comment:
IBM recommends to use a tag instead of using `latest` docker image.
If latest tag is used for docker image, then it is advised to use pull policy of always to get the latest available image.
This is more a recommendation geared towards promoting the chart to become a cloud pak. Not required at our initial pass.